### ISSUE TITLE
Running an alert displays an alert after every subsequent command

### ISSUE THREAD

**Issue by brandly** [callumacrae/tryregex#16]
Cool site! Since it looks like you're storing all `codeSoFar` and `eval`ing it every time, you get repetitive `alert`s if you ever run one. I'm not sure what the best solution is. Maybe `alert`, `confirm`, and `prompt` should be overridden in the `eval`ed context.

**Comment by callumacrae** [comment]
Yeah, I know about that. I don't know the best solution either, but it only seems to break when people try to break it so I don't really have much of a problem with that.
